descriptions/ntcurrentprocess.md

```diff
@@ -1,4 +1,7 @@
-This macro is [documented in Windows Driver Kit](https://learn.microsoft.com/en-us/windows-hardware/drivers/kernel/zwcurrentprocess). It is a native equivalent of the [`GetCurrentProcess`](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess) function and returns a pseudo-handle that grants `PROCESS_ALL_ACCESS` to the current process. You do not need to call `NtClose` on the returned handle.
+This macro is a native equivalent of the [`GetCurrentProcess`](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess) function. It returns a pseudo-handle that grants `PROCESS_ALL_ACCESS` to the current process. You do not need to call `NtClose` on the returned handle.
```

Comment:
Why did you remove the Windows Driver Kit link?

Reply:
Hmm, that wasn't intentional. I think I accidentally rolled back some changes in this and `NtCurrentThread` descriptions while adding the "Applicable to" section. I'll add a fix shortly